x-pack/plugins/observability_solution/investigate_app/server/lib/get_document_categories.ts

```diff
@@ -388,6 +424,9 @@ export const createCategorizationRequestParams = ({
   return {
     index,
     size: 0,
+    /* We occassionally end up with a  search_phase_execution_exception Caused by: illegal_argument_exception: 0 > -1
```

Comment:
This is a known error I reported here: https://github.com/elastic/elasticsearch/issues/112805